controllers/storageconsumer/storageconsumer_upgrade_controller.go

```diff
@@ -244,6 +234,10 @@ func (r *StorageConsumerUpgradeReconciler) reconcileStorageRequest(
 	if err := r.Client.Get(ctx, client.ObjectKeyFromObject(rbdStorageRequest), rbdStorageRequest); client.IgnoreNotFound(err) != nil {
 		return err
 	} else if rbdStorageRequest.UID != "" {
+		if err := r.reconcilePre4_19CephClients(ctx, rbdStorageRequest.UID, storageConsumer); err != nil {
+			return err
+		}
+
```

Comment:
Why did we move the code around? wouldn't this call make more sense in the same position of the original code?

Reply:
I had to move the code around as I need the storageRequest UID to list the ceph clients owned by the storage request